aws-lc-sys/builder/cmake_builder.rs

```diff
@@ -264,10 +264,16 @@ impl CmakeBuilder {
             emit_warning("!!!   Using pre-built NASM binaries   !!!");
             emit_warning("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!");
 
+            let script_name = if cfg!(target_os = "windows") {
```

Comment:
Doesn't this need to check for the _host_ OS in order to solve the issue of cross compiling linux -> windows?